plugin.video.embycon/resources/lib/websocket.py

```diff
@@ -894,8 +906,13 @@ def run_forever(self, sockopt=None, sslopt=None, ping_interval=0):
                     self._callback(self.on_message, data)                    
                     
                 except Exception as e:
-                    #print str(e.args[0])
-                    if "timed out" not in e.args[0]:
+                    found_timeout = False
+                    for arg in e.args:
+                        if isinstance(arg, str):
+                            print str(arg)
```

Comment:
The following change would have been enough to get CI green.
```suggestion
                            print(str(arg))
```

Reply:
But it is always something, I come to dread having to try to get the addon updated, it is not worth the stress and issues.

Reply:
Nothing against the project, it is just not for me, I am going to go back to a private repo, it is just so much easier to manage and update.